<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCLocationDlg</class>
 <widget class="QDialog" name="BCLocationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>快速开窗</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="BCLocationGroupWidget" name="m_pLocationGroupWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>430</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>网格定位</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="BCLocationSegmentationWidget" name="m_pLocationWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn1">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>整屏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>四分割</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn6">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>六分割</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn8">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>八分割</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn9">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>九分割</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn12">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>十二分割</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_pRadioBtn16">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>十六分割</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BCLocationSegmentationWidget</class>
   <extends>QWidget</extends>
   <header location="global">BCLocationSegmentationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BCLocationGroupWidget</class>
   <extends>QWidget</extends>
   <header>BCLocationGroupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
